recipes/pytest-monitor/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "pytest-monitor" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/CFMTech/pytest-monitor/archive/{{ name }}-{{ version }}.tar.gz
+  sha256: b4081347adcbc13c23eac3384a3dfa83a1c6301bb3aaae90e5588ec30a231ecd
+
+build:
+  number: 0
+  skip: True  # [py<35 or win]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - memory_profiler
+    - pytest
+    - psutil
+    - requests
```

Comment:
I don't think these are needed in build (or at install time). They are not compilers

Reply:
Thanks for spotting. I just got completely confused.